internal/providers/pluginfw/tfschema/customizable_schema.go

```diff
@@ -202,32 +202,12 @@ func (s *CustomizableSchema) SetReadOnly(path ...string) *CustomizableSchema {
 	return s
 }
 
-// ConvertToAttribute converts the last element of the path from a block to an attribute.
-// It panics if the path is empty, if the path does not exist in the schema, or if the path
-// points to an attribute, not a block.
-func (s *CustomizableSchema) ConvertToAttribute(path ...string) *CustomizableSchema {
-	if len(path) == 0 {
-		panic(fmt.Errorf("ConvertToAttribute called on root schema. %s", common.TerraformBugErrorMessage))
-	}
-	field := path[len(path)-1]
-
-	cb := func(attr BaseSchemaBuilder) BaseSchemaBuilder {
-		switch a := attr.(type) {
-		case BlockToAttributeConverter:
-			return a.ConvertBlockToAttribute(field)
-		default:
-			panic(fmt.Errorf("ConvertToAttribute called on invalid attribute type: %s. %s", reflect.TypeOf(attr).String(), common.TerraformBugErrorMessage))
-		}
-	}
-
-	// We have to go only as far as the second-to-last entry, since we need to change the parent schema
-	// by moving the last entry from a block to an attribute.
-	if len(path) == 1 {
-		s.attr = cb(s.attr)
-	} else {
-		navigateSchemaWithCallback(&s.attr, cb, path[0:len(path)-1]...)
-	}
-
+// ConfigureForSdkV2Migration modifies the underlying schema to be compatible with SDKv2. This method must
+// be called on all resources that were originally implemented using the SDKv2 and are migrated to the plugin
+// framework.
+func (s *CustomizableSchema) ConfigureForSdkV2Migration() *CustomizableSchema {
```

Comment:
I don't know why, but for me `ConfigureForSdkV2Migration` sends the message that it prepares it for a migration _towards_ SDKv2, how about something with "compatibility", like `MakeCompatibleWithSdkV2` or `ConfigureAsSdkV2Compatible`?